staff/staff_views_helper.py

```diff
@@ -189,6 +204,40 @@ def email_announcement(request, form, email_list):
     messages.add_message(request, messages.SUCCESS, "Recipients Successfully Emailed")
 
 
+def email_posted_form(request, form, email_list):
+    subject = form.cleaned_data.get("name")
+    print()
+    if form.cleaned_data.get("esign") is not None:
+        esign_link = form.cleaned_data.get("esign").template
+    else:
+        esign_link = None
+    print(form.cleaned_data.get("esign"))
+    print("esign link: ", esign_link)
+    msg_html = render_to_string(
+        "email_templates/post_form_email.html",
+        {
+            "subject": subject,
+            "message": form.cleaned_data.get("description"),
+            "from": DjangoUser.objects.get(id=request.user.id),
```

Comment:
I just created a forms@missionbit.com group we could use for this, not sure if the forms need to come from the particular instructor or staff member sending the request?

Reply:
It would be perfect if forms all came from forms@missionbit.com.  Presumably people will download/sign/reply to this email with completed forms, so it would make sense to warehouse all emailed form submissions within one.

Reply:
Creating issue to add this setting